addons/account/views/account_move_views.xml

```diff
@@ -621,7 +622,7 @@
                                         <!-- Displayed fields -->
                                         <field name="sequence" widget="handle"/>
                                         <field name="product_id"/>
-                                        <field name="name" widget="section_and_note_text"/>
+                                        <field name="name" widget="section_and_note_text" required="1"/>
```

Comment:
while you're at it fixing all small bugs, can you remove the widget="one2many_list" on field line_ids a few lines below:
<field name="line_ids" widget="one2many_list" ...

This widget does not exists and there is an error in console due to that.